api/appendices/deprecated_classes.adoc

```diff
@@ -0,0 +1,21 @@
+
+[[deprecated-classes]]
```

Comment:
I think perhaps "classes" is the wrong word here as it's an implementation detail - to the external API user they are "types"